plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -107,8 +107,11 @@ contract('PaymentExitGame - End to End Tests', ([_, richFather, bob]) => {
         this.erc20Vault = await Erc20Vault.new(this.framework.address);
         await this.erc20Vault.setDepositVerifier(erc20DepositVerifier.address);
 
-        await this.framework.registerVault(1, this.ethVault.address);
-        await this.framework.registerVault(2, this.erc20Vault.address);
+        await this.framework.registerVault(ETH_VAULT_ID, this.ethVault.address);
+        await this.framework.registerVault(ERC20_VAULT_ID, this.erc20Vault.address);
+
+        await this.framework.addExitQueue(ETH_VAULT_ID, ETH);
```

Comment:
Minor: consider add similar test as `Given ERC20 token added to the PlasmaFramework` instead of initiating here.
But I am fine without doing this because at one point we might want to change the setup here using our migration script result instead (so more real e2e) and our migration script might want to add ETH by default.

Reply:
That's what I thought, so I'll leave it as it is.